app/components/forms/admin/settings/payment-gateway-form.js

```diff
@@ -134,6 +153,10 @@ export default Component.extend(FormMixin, {
     return this.get('settings.omiseTestPublic') || this.get('settings.omiseLivePublic');
   }),
 
+  isCheckedAliPay: computed(function() {
```

Comment:
Ensure your computed property _watches_ the keys it depends on ...

Reply:
@niranjan94 Passed the keys into the computed property so that it is updated properly